relay/session.go

```diff
@@ -0,0 +1,102 @@
+package relay
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"time"
+)
+
+type Server struct {
+	vultisigRelay string
+}
+
+func NewServer(vultisigRelay string) *Server {
+	return &Server{
+		vultisigRelay: vultisigRelay,
+	}
+}
+
+func (s *Server) StartSession(sessionID string) error {
+	sessionURL := s.vultisigRelay + "/start/" + sessionID
+	body := []byte("[]")
+	bodyReader := bytes.NewReader(body)
+	resp, err := http.Post(sessionURL, "application/json", bodyReader)
+	if err != nil {
+		return fmt.Errorf("fail to start session: %w", err)
+	}
+	if resp.StatusCode != http.StatusCreated {
+		return fmt.Errorf("fail to start session: %s", resp.Status)
+	}
+	// session registered , and device can use this session id to join the session
+	return nil
+}
+
+func (s *Server) RegisterSession(sessionID string, key string) error {
+	sessionURL := s.vultisigRelay + "/" + sessionID
+	fmt.Println("Registering session with url: ", sessionURL)
```

Comment:
This is not production ready code , `fmt.Println` everywhere , if you need to log those messages , use the logging library we have in place, otherwise  remove it